### ISSUE TITLE
Pipeline's eval step fails due to offline website

### ISSUE THREAD

**Issue by racng** [cnobles/iGUIDE#88]
Pipeline's eval step failed because it failed to open the url "http://bushmanlab.org/assets/doc/allOnco_Feb2017.tsv".
I tried visiting the website but it is currently unresponsive.
Can these gene sets (allOnco_Feb2017, humanLymph) be added to this repository under an asset folder?

**Comment by cnobles** [comment]
Thanks for bringing this to my attention @racng, I have contacted the moderators for the site and let them know that it's down. Sometimes this happens, but the Bushman lab is always good about followup and has issues resolved like this shortly.

I searched my own data and also couldn't find a copy of the file. Some alternative files that may not contain the same info, but I like your suggestion. I will move a copy of this file (and the humanLymph.tsv) to part of the repo as soon as the site is up and running again, just so they are archived. I may request copies of the files be sent to me from their local servers if they are running into issues, but I don't want to distract them from solving the larger problem first.

**Comment by cnobles** [comment]
The Bushman Lab was able to restart the website with the reference files. I'm also working on releasing a new version with those files (and a few updated ones) saved to the repository as well. I'll be including a few other changes, so it may take a little bit, but if you would like to follow the progress, check the v1.1.1 branch.

**Comment by cnobles** [comment]
Resolved in 3cf58c74832b1956c5a32e2e08923894bb522c89, check for v1.1.1 release soon.

**Comment by marencc** [comment]
Hi! Many thanks for the support! 
I ran the pipeline two days ago and I am also getting: 

```
Error in download.file(input, tmpFile <- tempfile(), method = method,  : 
  cannot open URL 'http://bushmanlab.org/assets/doc/allOnco_Feb2017.tsv'
Calls: suppressMessages ... withCallingHandlers -> loadRefFiles -> <Anonymous> -> download.file
In addition: Warning message:
In download.file(input, tmpFile <- tempfile(), method = method,  :
  cannot open URL 'https://bushmanlab.org/assets/doc/allOnco_Feb2017.tsv': HTTP status was '404 Not Found'
Execution halted
```
I am on: 
```
iguide --version
iguide v1.1.1+4c8a25a
```

How can I resolve it? 
Many thanks, 

Maria

**Comment by cnobles** [comment]
Hi @marencc ,

I suspect that your config files have not been updated to reflect the use of the local files during processing. Check out the simulation config file which has been updated for an example. Below is a section that shows how to reference the utility files that are now part of the repo.

'''
# Post-processing
refGenes :
    file : "genomes/hg38.refSeq.ext.nomodel.rds"
    symbolCol : "name2"
oncoGeneList : 
    file : "genomes/allOnco_Feb2017.tsv"
    symbolCol : "symbol"
specialGeneList : 
    file : "genomes/humanLymph.tsv"
    symbolCol : "symbol"
'''

Let me know if that resolves the issue!

**Comment by marencc** [comment]
@cnobles Yes, I was "re-using" a previous version of the file, one I used a while ago.. My bad, sorry and many thanks!
